x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -116,7 +116,7 @@ export const allowedExperimentalValues = Object.freeze({
   /**
    * Enables new Knowledge Base Entries features, introduced in `8.15.0`.
    */
-  assistantKnowledgeBaseByDefault: false,
+  assistantKnowledgeBaseByDefault: true,
```

Comment:
We cannot set this value to `false` from dev.yml. If the intention is to be able to turn off the feature in case something goes wrong, you should reverse condition of the flag in the code and change the flag name to something like:

```
assistantKnowledgeBaseDisabled: false,
```

Make sense?